bitnami/reportserver/README.md

```diff
@@ -1,5 +1,9 @@
 # Bitnami package for ReportServer Community
 
+## Deprecation Notice
+
+The ReportServer container is no longer maintained by upstream and is now deprecated. This image will no longer be released in our catalog, but already released container images will still persist in the registries.
```

Comment:
```suggestion
The ReportServer container will no longer be released in our catalog, but already released container images will still persist in the registries.
```

In this case, the reason is not the asset deprecated upstream but an internal decision to keep the catalog fresh